custom_components/myskoda/translations/en.json

```diff
@@ -109,6 +165,9 @@
             }
         },
         "image": {
+            "render_light_3x": {
+                "name": "Status Render of Vehicle"
+            },
```

Comment:
This does not belong in this PR, please remove it

Reply:
sorry, it got mixed up with my test translation heh... already preparing readme update ;)